config/livewire.php

```diff
@@ -3,156 +3,158 @@
-    | cases like when hosting on Laravel Vapor, it could be set to a different value.
+    |---------------------------------------------------------------------------
+    | Navigate (SPA mode)
+    |---------------------------------------------------------------------------
     |
-    | Example: for Laravel Vapor, it would be "/tmp/storage/bootstrap/cache/livewire-components.php".
+    | By adding `wire:navigate` to links in your Livewire application, Livewire
+    | will prevent the default link handling and instead request those pages
+    | via AJAX, creating an SPA-like effect. Configure this behavior here.
     |
     */
 
-    'manifest_path' => null,
+    'navigate' => [
+        'show_progress_bar' => true,
+        'progress_bar_color' => '#2299dd',
+    ],
 
     /*
-    |--------------------------------------------------------------------------
-    | Back Button Cache
-    |--------------------------------------------------------------------------
-    |
-    | This value determines whether the back button cache will be used on pages
-    | that contain Livewire. By disabling back button cache, it ensures that
-    | the back button shows the correct state of components, instead of
-    | potentially stale, cached data.
+    |---------------------------------------------------------------------------
+    | HTML Morph Markers
+    |---------------------------------------------------------------------------
     |
-    | Setting it to "false" (default) will disable back button cache.
+    | Livewire intelligently "morphs" existing HTML into the newly rendered HTML
+    | after each update. To make this process more reliable, Livewire injects
+    | "markers" into the rendered Blade surrounding @if, @class & @foreach.
     |
     */
 
-    'back_button_cache' => false,
+    'inject_morph_markers' => true,
 
     /*
-    |--------------------------------------------------------------------------
-    | Render On Redirect
-    |--------------------------------------------------------------------------
+    |---------------------------------------------------------------------------
+    | Pagination Theme
+    |---------------------------------------------------------------------------
     |
-    | This value determines whether Livewire will render before it's redirected
-    | or not. Setting it to "false" (default) will mean the render method is
-    | skipped when redirecting. And "true" will mean the render method is
-    | run before redirecting. Browsers bfcache can store a potentially
-    | stale view if render is skipped on redirect.
+    | When enabling Livewire's pagination feature by using the `WithPagination`
+    | trait, Livewire will use Tailwind templates to render pagination views
+    | on the page. If you want Bootstrap CSS, you can specify: "bootstrap"
     |
     */
 
-    'render_on_redirect' => false,
-
+    'pagination_theme' => 'tailwind',
```

Comment:
We don't have any components that are using pagination yet so I skipped that because it is possible to use [bootstrap](https://livewire.laravel.com/docs/pagination#using-bootstrap-instead-of-tailwind) but it is the [latest version](https://github.com/livewire/livewire/blob/main/src/Features/SupportPagination/views/bootstrap.blade.php) so I would have to downgrade it to bootstrap v3's syntax.

I figured I'd skip that until we actually need pagination but maybe I should just include it so we don't have to worry about it in the future...

Gonna move this back to draft and include that...

Reply:
Yeah, you're right - we don't actually use Laravel pagination anywhere. You could probably just let it sit, since v8 will either be filament, OR will be just be more of our own API, which doesn't use UI pagination.

Reply:
That's a good point...gonna leave this alone and just focus on the conflicts.